pkg/cli/alpha/config-gen/controller-gen-filter.go

```diff
@@ -64,7 +64,10 @@ func (cgr ControllerGenFilter) Filter(input []*yaml.RNode) ([]*yaml.RNode, error
 
 	// set the directory
 	b := bufferedGenerator{}
-	rt, _ := gens.ForRoots(cgr.Spec.CRDs.SourceDirectory) // ignore the spurious error
+	rt, err := gens.ForRoots(cgr.Spec.CRDs.SourceDirectory)
+	if err != nil {
+		return nil, errors.WrapPrefixf(err, "failed to produce generator's Runtime against %s", cgr.Spec.CRDs.SourceDirectory)
+	}
```

Comment:
@pwittrock could you please give a hand here? Why have we are ignoring these errors before?